test/jdk/java/awt/event/MouseEvent/ClickDuringKeypress/ClickDuringKeypress.java

```diff
@@ -22,115 +22,100 @@
  */
 
 /*
-  @test 1.2 98/08/05
   @key headful
   @bug 4515763
   @summary Tests that clicking mouse and pressing keys generates correct amount of click-counts
-  @author andrei.dmitriev: area=awt.mouse
   @run main ClickDuringKeypress
 */
 
-/**
- * ClickDuringKeypress.java
- *
- * summary:
- */
-
-import java.applet.Applet;
-import java.awt.*;
-import java.awt.event.*;
-
-public class ClickDuringKeypress implements MouseListener
- {
+import java.awt.AWTException;
+import java.awt.EventQueue;
+import java.awt.Frame;
+import java.awt.Robot;
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.event.WindowEvent;
+
+public class ClickDuringKeypress implements MouseListener {
    //Declare things used in the test, like buttons and labels here
    final static int CLICKCOUNT = 10;
    final static int DOUBLE_CLICK_AUTO_DELAY = 10;
-   volatile int lastClickCount = 0;
-   volatile boolean clicked = false;
-   volatile boolean ready = false;
-
-   Frame frame;
-   Robot robot;
-
-   public void init()
-    {
-      //Create instructions for the user here, as well as set up
-      // the environment -- set the layout manager, add buttons,
-      // etc.
-
+   static volatile int lastClickCount = 0;
+   static volatile boolean clicked = false;
+   static volatile boolean ready = false;
+
+   static volatile Frame frame;
+   static volatile Robot robot;
+   static volatile ClickDuringKeypress clicker;
```

Comment:
```suggestion
   static final ClickDuringKeypress clicker = new ClickDuringKeypress();
```
`clicker` can be made final

Reply:
ok